myia/api.py

```diff
@@ -1,10 +1,20 @@
 """User-friendly interfaces to Myia machinery."""
 
+import numpy as np
+
+from myia import dtype
 import inspect
 
 from .abstract import MyiaTypeError, from_value
 from .pipeline import standard_pipeline
-from .utils import keyword_decorator
+from .utils import keyword_decorator, overload
+
+from .abstract import AbstractTuple, AbstractList, AbstractClass, \
+    AbstractArray, TYPE, AbstractScalar, AbstractUnion
+
+from .abstract.infer import ArrayWrapper
+
+from myia.compile.backends import load_backend
```

Comment:
Import ArrayWrapper from .abstract, use a relative import for compile, and compress the whitespace.